x-pack/test/api_integration/apis/endpoint/data_stream_helper.ts

```diff
@@ -30,10 +29,6 @@ export async function deleteMetadataStream(getService: (serviceName: 'es') => Cl
   await deleteDataStream(getService, metadataIndexPattern);
 }
 
-export async function deleteMetadataMirrorStream(getService: (serviceName: 'es') => Client) {
```

Comment:
Is the hope to remove the `metadata_mirror` from the package after these changes get merged?

Reply:
Yes, once merged. I will clean up the endpoint artifact.